### PR TITLE
update day-plan for week 4

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

Module:
Week(s):

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: content/js1/sprints/4/day-plan/index.md [hunk 1]
```diff
@@ -6,16 +6,17 @@ menu_level = ['sprint']
...
-name="pd-placeholder"
+name= "Asking good technical questions"
+src= "https://cyf-pd.netlify.app/blocks/asking-good-technical-questions/readme/"
 [[blocks]]
 name="morning-break"
 [[blocks]]
-name="Handling undefined"
-src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/undefined"
+name="⌚ Time string"
+src="https://github.com/CodeYourFuture/CYF-Workshops/blob/main/time-string/readme.md"
```
Comment (SallyMcGrath) [inline content/js1/sprints/4/day-plan/index.md:18]:
```suggestion
src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/time-string"
```

Comment (SallyMcGrath) [reply content/js1/sprints/4/day-plan/index.md:18]: Try that? The build is failing on this call

https://app.netlify.com/sites/cyf-curriculum/deploys/65031ed10d8ecd000841dab7

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Failing build bad link

File: content/js1/sprints/4/day-plan/index.md (hunk 1 above)
Comment (Dedekind561) [reply content/js1/sprints/4/day-plan/index.md]: just removed the filename from the path

**Review by SallyMcGrath — APPROVED** [review]
LGTM

**Pull request merged**